tvb/simulator/plot/mayavi_tools.py

```diff
@@ -101,14 +102,20 @@ def surface_timeseries(surface, data, step=1):
             k += step
             yield
         mlab.show()
-        #--------------------------------------------------------------------------#
+        # --------------------------------------------------------------------------#
+
 
+    # TODO: Make, posssibly with a wrapper function, to work directly with
+    # SurfacePattern object... Inner function name plot_surface
+    def surface_p(plot_surface):
```

Comment:
How will this be used exactly?
Can you write an example in the documentation comment on this methond?

Reply:
Yes I will love todo that